.github/workflows/conformance-runtime.yaml

```diff
@@ -322,6 +321,16 @@ jobs:
             export GOTEST_FORMATTER="/root/go/bin/go-junit-report -set-exit-code -iocopy -out test/runtime.xml"
             make tests-privileged NO_COLOR=1
 
+      - name: Debug failure on VM
+        if:  ${{ !success() && (matrix.focus == 'agent' || matrix.focus == 'datapath') }}
```

Comment:
Could you explain the condition here?

Reply:
Sure, the `${{ matrix.focus == 'privileged' }}` is not running Cilium service, it's only running unit tests.

Reply:
Ah, I see. Should the condition be inverted to make that clear?

Reply:
I'm not following, you mean `if:  ${{ !success() && matrix.focus != 'privileged' }}` would make it more clear than `(matrix.focus == 'agent' || matrix.focus == 'datapath')`?

Reply:
Yes, because it makes it clear this is specific to the privileged test. It's not so much that we only want to run this for agent and datapath, it's more that we don't want to run it for the privileged test. Said some other way, if we add a suite for hubble, we would likely want to run it there.

That said, nothing beats a small comment to clarify code.

Reply:
> Yes, because it makes it clear this is specific to the privileged test. It's not so much that we only want to run this for agent and datapath, it's more that we don't want to run it for the privileged test. Said some other way, if we add a suite for hubble, we would likely want to run it there.

Sure, but the same argument can be used the other way around. What if we add a suite for hubble but we don't want it to run it there?

I've added the comment to make it clear.